stencil-library/src/components/form/form-control-number-masked.tsx

```diff
@@ -37,9 +36,8 @@ export class NumberInputMasked {
   }
 
   componentDidLoad() {
-    const inputElement = this.el.shadowRoot.querySelector('input');
-    if (inputElement && this.mask) {
-      this.imask = IMask(inputElement, {
+    if (this.textInput && this.mask) {
```

Comment:
Does this condition need the `&& this.mask`? I noticed that the other form controls don't check for it (or maybe those need it?)

Reply:
Actually, it's just an extra check in case the component don't receive any mask so the logic won't break. The other components don't have the mask prop.

Reply:
Ahh, okay. Thanks for the clarification!